src/services/helpers.ts

```diff
@@ -117,3 +124,44 @@ export function addPagersFn<T>(
     next,
   };
 }
+
+export function addCustomPropertiesToConfig(
```

Comment:
I know this function can be written shorter but I went for super expressive here 😄